eng/scripts/GenerateAndBuildLib.ps1

```diff
@@ -0,0 +1,886 @@
+#Requires -Version 7.0
```

Comment:
remove this

Reply:
Hi Arthur, I didn't add this line of code, it came in from the forked repo azure-sdk-for-net.